rdkit/Chem/Suppliers/DbMolSupplier.py

```diff
@@ -78,15 +78,13 @@ def _BuildMol(self, data):
       elif self.molFmt == 'PKL':
         newM = Chem.Mol(str(molD))
     except Exception:
-      import traceback
       traceback.print_exc()
       newM = None
     else:
       if newM and self.transformFunc:
         try:
           newM = self.transformFunc(newM, data)
         except Exception:
-          import traceback
```

Comment:
I don't think it makes sense to have the traceback import here. It's a pretty basic module.